rero/rero-mef
# feat: implement the additional transformation

Signed-off-by: Gianni Pante <gianni.pante@rero.ch>
Co-author : Nicolas Labat <nicolas.labat@rero.ch>
Co-author : Aly Badr <aly.badr@rero.ch>

To test:
pip install -e .[all]
./scripts/bootstrap
./scripts/setup
./run-tests

Display the sources (bnf, gnd, rero) records:
http://localhost:5000/api/bnf/?q=&prettyprint=1&size=100
http://localhost:5000/api/gnd/?q=&prettyprint=1&size=100
http://localhost:5000/api/rero/?q=&prettyprint=1&size=100
and make sure the identifier of person field is a URI
